client/helpers/scenes.ts

```diff
@@ -268,6 +245,24 @@ export async function createSceneItems({
   parallelism,
   sceneId,
 }: CreateSceneItemsReq): Promise<CreateSceneItemsRes> {
+  // set ordinals based on request order
+  const reqMap: Map<string, CreateSceneItemRequest[]> = new Map();
+  reqMap.set('', []);
+  createSceneItemReqs.forEach((req) => {
+    const reqParent =
+      req.data.attributes.parent ??
+      req.data.relationships.parent?.data.id ??
+      '';
+    if (!reqMap.has(reqParent)) {
+      reqMap.set(reqParent, []);
+    }
+    const siblings = reqMap.get(reqParent);
+    if (req.data.attributes.ordinal === undefined) {
```

Comment:
It's preferred to not use strict equals (`===`) when checking for emptiness in JS, `ordinal = null` will fail this check. Prefer `req.data.attributes.ordinal == null`, which'll handle both `null` and `undefined`.